rotkehlchen/data_import/importers/bitstamp.py

```diff
@@ -0,0 +1,159 @@
+                sequence_index=2,
+                timestamp=timestamp,
+                location=Location.BITSTAMP,
+                asset=fee_currency,
+                balance=Balance(
+                    amount=fee_amount,
+                    usd_value=ZERO,
+                ),
+                notes=f'Fee of {fee_amount} {fee_currency} as the result of a trade on Bitstamp',
+                event_type=HistoryEventType.TRADE,
+                event_subtype=HistoryEventSubType.FEE,
+            )
+            self.add_history_events(write_cursor, [
+                spend_trade_event,
+                receive_trade_event,
+                fee_event,
+            ])
+        elif csv_row['Type'] == 'Deposit' or csv_row['Type'] == 'Withdrawal':
+            amount = deserialize_asset_amount(amount)
+            asset = asset_from_bitstamp(amount_symbol)
+            transaction_type = csv_row['Type']
+            movement_event = HistoryEvent(
+                event_identifier=event_identifier,
+                sequence_index=0,
+                timestamp=timestamp,
+                location=Location.BITSTAMP,
+                asset=asset,
+                balance=Balance(
+                    amount=amount,
+                    usd_value=ZERO,
+                ),
+                notes=f'{transaction_type} of {amount} {asset} on Bitstamp',
+                event_type=HistoryEventType.DEPOSIT if transaction_type == 'Deposit' else HistoryEventType.WITHDRAWAL,  # noqa: E501
+                event_subtype=HistoryEventSubType.NONE,
+            )
+            self.add_history_events(write_cursor, [movement_event])
+
+    def _import_csv(self, write_cursor: DBCursor, filepath: Path, **kwargs: Any) -> None:
+        """
+        Import trades from bitstamp.
+        May raise:
+        - InputError if one of the rows is malformed
+        """
+        with open(filepath, encoding='utf-8-sig') as csvfile:
+            data = csv.DictReader(csvfile)
+            for row in data:
+                try:
+                    self._consume_bitstamp_transaction(write_cursor, row, **kwargs)
+                except UnknownAsset as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'During Bitstamp CSV import found action with unknown '
+                        f'asset {e.identifier}. Ignoring entry',
+                    )
+                    continue
+                except DeserializationError as e:
+                    self.db.msg_aggregator.add_warning(
+                        f'Deserialization error during Bitstamp CSV import. '
+                        f'{e!s}. Ignoring entry',
+                    )
+                    continue
+                except KeyError as e:
+                    raise InputError(f'Could not find key {e!s} in csv row {row!s}') from e
+                except ValueError as e:
+                    raise InputError(f'Could not parse some values {e!s} in csv row {row!s}') from e  # noqa: E501
```

Comment:
1. You don't need to continue. There is nothing to continue for.
2. Why is a ValueError a fatal error skipping all other lines? Same question for KeyError.

```suggestion
            for row in data:
                try:
                    self._consume_bitstamp_transaction(write_cursor, row, **kwargs)
                except UnknownAsset as e:
                    self.db.msg_aggregator.add_warning(
                        f'During Bitstamp CSV import found action with unknown '
                        f'asset {e.identifier}. Ignoring entry',
                    )
                except DeserializationError as e:
                    self.db.msg_aggregator.add_warning(
                        f'Deserialization error during Bitstamp CSV import. '
                        f'{e!s}. Ignoring entry',
                    )
                except KeyError as e:
                    raise InputError(f'Could not find key {e!s} in csv row {row!s}') from e
                except ValueError as e:
                    raise InputError(f'Could not parse some values {e!s} in csv row {row!s}') from e  # noqa: E501
```